ant-cli/src/commands.rs

```diff
@@ -65,6 +68,8 @@ pub enum FileCmd {
         addr: String,
         /// The destination file path.
         dest_file: String,
+        /// Optionally specify the quorum for the verification of the download.
+        read_quorum: Option<ResponseQuorum>,
```

Comment:
Could we have a CI test proving this is use-able in the CLI? I have no idea how to provide this argument in the CLI.